app/config/events.php

```diff
@@ -190,6 +190,9 @@
         ],
         'update' => [
             '$description' => 'This event triggers when a bucket is updated.',
```

Comment:
```suggestion
            '$description' => 'This event triggers when a team is updated.',
```
Don't we wanna update this description as well 😅

Reply:
Similarly, some of the other descriptions in the teams events use `bucket`